buildslave/bootstrap.sh

```diff
@@ -17,6 +17,11 @@ apt-get install -y ntp
 apt-get install -y dpkg-dev debhelper fakeroot
 apt-get install -y libexpat1-dev
 
+dpkg --add-architecture i386
+apt-get update -qy
+apt-get install -qy wine-development:i386 mingw-w64
```

Comment:
Do we need *wine*? Isn't that just for the *candle & light* things we don't support here anyway?

Reply:
I think you are right, but it may be worth to keep here and try to work towards supporting it?

Reply:
Works for me.